src/sys/om.c

```diff
@@ -1357,7 +1358,8 @@ OMCamera* omGObjSetCamera(GObj* obj) {
     obj->data.cam = camera;
     camera->obj = obj;
     func_80007CBC(&camera->vp);
-    if (FALSE) {} // required to match
+    if (FALSE) {
```

Comment:
These probably would look less weird if we just put the `// required to match` on the line above